### PR TITLE
Example: Cypress

### PR DESCRIPTION
**Pull request by tarang9211**

### CODE REVIEW

File: apps/next-cypress/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (Noviny) [inline apps/next-cypress/tsconfig.json]: Interesting learning thing - in a monorepo, we should only have a `tsconfig` at the repo root - and if absolutely necessary a lower down one that extends that.

File: cypress.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Noviny) [inline cypress.json]: This being in the root is a little odd to me.

File: apps/next-cypress/tsconfig.json (hunk 1 above)
Comment (mitchellhamilton) [reply apps/next-cypress/tsconfig.json]: sad thing: Next always generates a tsconfig for you

Comment (Noviny) [reply apps/next-cypress/tsconfig.json]: BOOOOOO

What even?

We can't even gitignore them because sometimes we need them right? :(

Comment (tarang9211) [reply apps/next-cypress/tsconfig.json]: So in the latest commit - `707e2d2` I deleted most of the auto generated Cypress tests and kept only one. I think the `.tsconfig` can be fixed, that shouldn't be an issue.

**Comment by tarang9211** [comment]
@Noviny Also, docs for this is remaining.

**Comment by mitchellhamilton** [comment]
List of things left to do in this PR:
- [ ] Use https://www.npmjs.com/package/start-server-and-test to start next and cypress at the same time and put that in the `cypress-ci` script
- [ ] Figure out if we can delete all the extra files cypress created 
- [ ] Resolve the conflicts with master

**Comment by tarang9211** [comment]
@mitchellhamilton I've updated the PR and modified the scripts. I've also disabled videos and plugins for cypress. I think this should be good?

File: package.json [hunk 3]
```diff
@@ -18,8 +18,8 @@
     "release": "yarn build && changeset publish",
     "test": "jest",
     "clean": "manypkg exec rm -rf node_modules && manypkg exec rm -rf dist && rm -rf node_modules",
-    "start:next": "yarn workspace @monorepo-starter/next-app run dev",
-    "start:server": "yarn workspace @monorepo-starter/graphql-api run start",
+    "start:next": "yarn manypkg run next-cypress start",
```
Comment (mitchellhamilton) [inline package.json]:
```suggestion
    "start:next": "yarn manypkg run next-app start",
```

**Pull request merged**